src/applications/hca/tests/unit/containers/ConfirmationPage.unit.spec.js

```diff
@@ -39,79 +41,66 @@ describe('hca ConfirmationPage', () => {
       dispatch: () => {},
     },
   });
-
-  context('when the page renders', () => {
-    it('should render success alert & application summary', () => {
-      const { mockStore } = getData({});
-      const { container } = render(
-        <Provider store={mockStore}>
-          <ConfirmationPage />
-        </Provider>,
-      );
-      const selectors = {
-        page: container.querySelector('.hca-confirmation-page'),
-        alert: container.querySelector('.hca-success-message'),
-        summary: container.querySelector('va-summary-box'),
-      };
-      expect(selectors.page).to.not.be.empty;
-      expect(selectors.summary).to.exist;
-      expect(selectors.alert).to.exist;
+  const subject = ({ mockStore }) => {
+    const { container } = render(
+      <Provider store={mockStore}>
+        <ConfirmationPage />
+      </Provider>,
+    );
+    const selectors = () => ({
+      page: container.querySelector('.hca-confirmation-page'),
+      alert: container.querySelector('.hca-success-message'),
+      summary: container.querySelector('va-summary-box'),
+      veteranName: container.querySelector('.hca-veteran-fullname'),
+      applicationDate: container.querySelector('.hca-application-date'),
     });
+    return { container, selectors };
+  };
+
+  it('should render success alert & application summary when the page renders', () => {
+    const { mockStore } = getData({});
+    const { selectors } = subject({ mockStore });
+    const { alert, page, summary } = selectors();
+    expect(page).to.not.be.empty;
+    expect(summary).to.exist;
+    expect(alert).to.exist;
   });
 
-  context('when the submission is parsed', () => {
-    it('should not render application date container when there is no response data', () => {
-      const { mockStore } = getData({});
-      const { container } = render(
-        <Provider store={mockStore}>
-          <ConfirmationPage />
-        </Provider>,
-      );
-      const selector = container.querySelector('.hca-application-date');
-      expect(selector).to.not.exist;
-    });
+  it('should not render application date container when there is no response data', () => {
+    const { mockStore } = getData({});
+    const { selectors } = subject({ mockStore });
+    const { applicationDate } = selectors();
+    expect(applicationDate).to.not.exist;
+  });
 
-    it('should render application date container when there is response data', () => {
-      const { mockStore } = getData({
-        submission: {
-          response: {
-            timestamp: 1262322000000,
-            formSubmissionId: '3702390024',
-          },
+  it('should render application date container when there is response data', () => {
```

Comment:
@IGallupSoCo , here is the fix for the specific test you referenced. We are now specifically declaring that timestamp as UTC and then just testing for the format in the test itself. This should remove the test being a day off based on the local machine running the test.